index.html

```diff
@@ -76,11 +76,11 @@ <h2>
         Introduction
       </h2>
       <p>
-        The Screen Orientation API allows web applications that would be
-        negatively impacted by users changing the orientation of their device
-        to block any changes. It also provides the current screen orientation
-        and any orientation changes so all web applications can display
-        correctly.
+        The Screen Orientation API provides the current screen orientation and
```

Comment:
The thing we need to be careful of is making claims about being "correct". I'd also like to make sure the intro covers: angle, type, event - and why... and also that this is not a substitute for CSS (the locking aspect is the most important with that regards... CSS can't lock, but CSS does the actual content adaptation). Most of that is in what you wrote, but how about (feel free to change!): 

>  For web applications, the Screen Orientation API exposes the type and angle of a device's current screen orientation, as well as the ability to be notified if the device's orientation changes. This allows web applications to programmatically adapt the user experience for many possible screen orientations (in concert with CSS). The API also affords web applications with the ability to lock the screen to a particular orientation, which can provide for an optimal user experience in some types of applications. This is useful in applications like computer games, where the user can physically rotate the device as a means of user input but the screen orientation itself mustn't  change.

Reply:
This is now edited, please review the intro as a whole, think it's all ready